index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
+  <title>Awesome Books</title>
+</head>
+<body>
+  <header>
+    <div class="top-nav">
+      <span>Awesome Books</span>
+      <nav>
+        <ul>
+          <li><a href="#" id="books-link">List</a></li>
+          <li><a href="#" id="add-book-link">Add Book Form</a></li>
+          <li><a href="#" id="contact-link">Contact</a></li>
+        </ul>
+      </nav>
+
+    </div>
+    <div class="date-container" id="date">
+        
+    </div>
+  </header>
+
+  <main>
+    <section id="books-section" >
+      <h2 class="book-title">All Awesome Books</h2>
+      <div id="books-list" class="book-stack">
+        
+      </div>
+    </section>
+
+    <section id="add-book-section" >
+      <h2 class="book-title">Add Book Form</h2>
+      <form id="add-book-form">
+        <div>
+          <label for="title">Title:</label>
+          <input type="text" id="title" name="title" required>
+        </div>
+        <div>
+          <label for="author">Author:</label>
+          <input type="text" id="author" name="author" required>
+        </div>
+        <div>
+          <label for="genre">Genre:</label>
+          <input type="text" id="genre" name="genre" required>
+        </div>
+        <div>
+          <label for="year">Year:</label>
+          <input type="number" id="year" name="year" required>
+        </div>
+        <button type="submit">Add Book</button>
+      </form>
```

Comment:
- [ Required ] I think it would be better if your design for the Add Book section is similar to the design on the project wireframe for this milestone. Currently, your design varies from the wireframe in the following ways.
  
  -  [ ] The styling of elements in your Add Book form is not consistent with the project wireframe. Please check the `padding`, `margins`, and `alignment` of the form elements. The `Add` button should be aligned to the `right` of the form.
   - [ ] The form is not centered in the middle of the page, kindly center it. You can do this using the flex property `justify-content: center `

## Your design
![Screenshot from 2023-07-07 02-18-02](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/16392046/5bc51460-c39f-4562-81cb-e92ac0c56aea)

## Project wireframe
![awesome_books_full_website](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/16392046/cd37a031-7790-48cd-a731-d6ee1ee90aea)

Reply:
Done!!!
![image](https://github.com/lerfast/Awesome-books-a-complete-website-with-navigation/assets/111823606/9b4e6f17-62f3-40d7-9bb3-2fdf3ec1df74)